runner/runners/queue.go

```diff
@@ -409,6 +413,7 @@ func (c *QueueController) loop() {
 			case abortReq:
 				st, err := c.abort(r.runID)
 				r.resultCh <- result{st, err}
+				idleLatency.Time()
```

Comment:
Does this work without this line in the abort handler here? From what I can tell even in case of abort, we should still hit the normal <-watchCh case below (in fact the abort call here effectively measures when we've signaled the real run to abort by closing c.runningAbort - does not guarantee the run has really exited).

Reply:
Yes, agree that the watchCh will receive the signal with AbortStatus and it will anyway reset the current state. Removing this redundant line, thanks!